jbmc/regression/jbmc-strings/StringFormatSpecial/test-percent-length-pass.desc

```diff
@@ -0,0 +1,6 @@
+CORE symex-driven-lazy-loading-expected-failure
+Test.class
+--function Test.percentLength --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar` --property "java::Test.percentLength:()Ljava/lang/String;.assertion.2"
+^EXIT=0$
+^SIGNAL=0$
+line 16 assertion at file Test.java .*: SUCCESS
```

Comment:
Could you make both `-fail` and `-pass` checks in the same test? By not passing the `--property` flag?

Reply:
I could, but it would be less robust, as we wouldn't be able to check the EXIT value.